src/runtime/composables/useDirectusFiles.ts

```diff
@@ -33,7 +35,7 @@ export const useDirectusFiles = () => {
     fileId: string,
     options?: DirectusThumbnailOptions
   ): string => {
-    const url = new URL(`${directusUrl}assets/${fileId}`)
+    const url = new URL(`${directusUrl}/assets/${fileId}`)
```

Comment:
Maybe using `joinURL` method from https://github.com/unjs/ufo (which is installed with nuxt) would cover all case here

Reply:
Good idea :smile:! Does that mean `ufo` should be added as a `peerDependency` then?

Reply:
Im not really sure.

Reply:
You can add them as dependency honestly, it's very small